episodes/15-coding-conventions.md

```diff
@@ -657,13 +671,10 @@ Functions:
 ...
 ```
 
-The docstring for a function or a module
-is returned when calling the `help` function and passing its name -
-for example from the interactive Python console/terminal available from the command line
-or when rendering code documentation online
-(e.g. see [Python documentation](https://docs.python.org/3.11/library/index.html)).
-PyCharm also displays the docstring for a function/module
-in a little help popup window when using tab-completion.
+The docstring for a function or a module is returned when calling the `help` function and passing its name - 
```

Comment:
@anenadic - the long lines I referred to are in the source code for the page. This is an extreme example - but illustrates the differences in line length. Really we want to keep the line length short, as in the original 660 + 661 lines, so that when changes are made these are highlighted as close to the change as possible.

I'd recommend making sure that you have 'soft wrap' switched off in your text editor - this will highlight for you which lines are too long, as you'll have to scroll to read them (see 675 and 677 in the new code for an example of this ;-) )

Reply:
Ah, got you. Think we may have to fix that across the material too.

Reply:
After discussion at the maintainers meeting I'm going to retract this suggestion - if we do need to keep sentences on single lines to facilitate translation tooling then let's do this.